src/vs/workbench/parts/extensions/electron-browser/extensionsViewlet.ts

```diff
@@ -69,6 +70,18 @@ const SearchBuiltInExtensionsContext = new RawContextKey<boolean>('searchBuiltIn
 const RecommendedExtensionsContext = new RawContextKey<boolean>('recommendedExtensions', false);
 const DefaultRecommendedExtensionsContext = new RawContextKey<boolean>('defaultRecommendedExtensions', false);
 const GroupByServersContext = new RawContextKey<boolean>('groupByServersContext', false);
+const viewIdNameMappings: { [id: string]: string } = {
+	'extensions.listView': localize('marketPlace', "Marketplace"),
```

Comment:
Use consistent quoting style. For example always prefer single quotes over double quotes.

Reply:
That is true, but not when we localize strings. If we use single quotes on the string that is to be localized, then the localization doesnt work.

We have an exception on our pre check in hook for this. If double quotes were used elsewhere, then I wouldnt have been able to commit in the first place

Reply:
You can still prefer single quotes first and fallback to backtick if string includes single quotes.

Reply:
The rest of the file is using double quotes, so I'll keep so for consistency's sake